integreat_compass/core/settings.py

```diff
@@ -40,6 +40,7 @@
     "django.contrib.contenttypes",
     "django.contrib.sessions",
     "django.contrib.messages",
+    "livereload",
```

Comment:
At the moment, this integration is not working yet, so I suggest to either remove it or fix it.

![livereload](https://github.com/digitalfabrik/integreat-compass/assets/16021269/14af3306-c750-410e-870a-87146fbd498b)

Also, I'd suggest to add it to the installed apps only when the corresponding package is installed, otherwise this will raise an error in production mode where the dev dependencies are not installed.